validator/validator-main.protoascii

```diff
@@ -509,6 +509,25 @@ tags: {
     dispatch_key: NAME_VALUE_DISPATCH
   }
 }
+# AMP & AMP4ADS metadata, name=amp-experiments-opt-out
+# https://github.com/lannka/amphtml/blob/master/tools/experiments/README.md
```

Comment:
Plans to update the README with how to opt out?

Reply:
Yup, just didn't know if you wanted that in this PR or in a separate.

Reply:
Probably keep the documentation with whatever changes will actually implement this new tag.